lib/util.js

```diff
@@ -3,9 +3,12 @@
  */
 'use strict';
 
-const _ = require('lodash');
 const config = require('./config');
 const delay = require('delay');
+const lodashGet = require('lodash.get');
+const lodashSet = require('lodash.set');
+const lodashTemplate = require('lodash.template');
+const lodashToPath = require('lodash.topath');
```

Comment:
if the deprecation is correct, it looks like upgrading `lodash` then using:

```js
const lodashToPath = require('lodash/topath');
```

might be better.